examples/workflows/private-ai-foundation/steps/01/variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "vcenter_username" {
+  description = "Username used to authenticate against the vCenter Server"
+}
+
+variable "vcenter_password" {
+  description = "Password used to authenticate against the vCenter Server"
+}
+
+variable "vcenter_server" {
+  description = "FQDN or IP Address of the vCenter Server"
+}
```

Comment:
We should mention that this is the mgmt domain or workload VC, respectively. Also applies to the other variables.tf.

Reply:
Done!